src/main/core/Manager/MailManager.php

```diff
@@ -267,4 +272,31 @@ public function getMailerFrom()
 
         return $this->config->getParameter('help.support_email');
     }
+
+    public function sendRequestToDPO(UserInterface $user)
+    {
+        if ($this->isMailerAvailable()) {
+            $name = $user->getFullName();
+            $idUser = $user->getId();
+            $dpoEmail = $this->config->getParameter('privacy.dpo.email');
+            $locale = $user->getLocale();
+
+            $subject = $this->translator->trans('account_deletion.subject', [], 'privacy', $locale);
+            $body = $this->translator->trans('account_deletion.body', ['%name%' => $name, '%id%' => $idUser], 'privacy', $locale);
+            $fromEmail = $this->config->getParameter('mailer.from');
+
+            $body = $this->templateManager->getTemplate('email_layout', ['content' => $body], $locale);
+
+            $message = new Message();
+            $message->subject($subject);
+            $message->from($fromEmail);
+            $message->body($body);
+            $message->to($user->getEmail());
+            $message->replyTo($dpoEmail);
+
+            return $this->mailer->send($message);
+        }
```

Comment:
tu dois réutiliser la méthode `$this->send(...)` ici (et enlever les lignes 286 à 296)